clientHelpers.go.tmpl

```diff
@@ -76,7 +76,7 @@ export class {{$error.Name}}Error extends WebrpcError {
     name: string = '{{$error.Name}}',
     code: number = {{$error.Code}},
     message: string = `{{$error.Message}}`,
-    status: number = 0,
+    status: number = {{$error.Status}},
```

Comment:
Yes, this makes sense. Or, should we get the actual status from the fetch response object?